README.md

```diff
@@ -52,8 +52,25 @@ See [Command Line Interface](https://devexpress.github.io/testcafe/documentation
 
 > We are in the process of filling this repository with more examples.
 
-The examples are organized in the following sections:
-
-* [Inject Custom Client Scripts](examples/client-scripts/)
-* [Access Element Properties](examples/element-properties/)
-* [Manipulate the Tested Page](examples/page-manipulation/)
+The repository includes the following examples:
+
+* [Find Element By Trimmed Text](examples/find-element-by-trimmed-text)
+* [Use XPath Selectors](examples/use-xpath-selectors)
+* [Access Element Properties](examples/element-properties)
+* [Iterate Over Table Rows](examples/iterate-over-table-rows)
+* [Test \<select\> Elements](examples/test-select-elements)
+* [Page Reload](examples/page-manipulation)
+* [Extract Code to Helpers](examples/extract-code-to-helpers)
+* [Create Data-Driven Tests](examples/create-data-driven-tests)
+* [Use Page Model](examples/use-page-model)
+* [Inject Custom Client Scripts](examples/client-scripts)
+* [Import Third-Party Modules](examples/import-third-party-modules)
+* [Mock Date](examples/mock-date)
+* [Mock Geolocation API](examples/mock-geolocation-api)
+* [Wait Until an Element Property Has a Specific Value](examples/wait-for-element-property-value)
+* [Wait For File Download](examples/wait-for-file-download)
+* [Access Element Properties](examples/element-properties)
+
+Dedicated examples either run in one of the browsers (Chrome or Firefox) or require additional launch arguments. Launch those examples separately.
```

Comment:
I think we can come up with a better word than 'dedicated'. I don't think we can come up with a *good* word (because the idea to separate examples with arguments has nothing to do with semantics) but it should be better than 'dedicated'.

`examples-with-args`, `arg-dependent`, `config-sensitive`? A combination of those?

Reply:
> detached-examples